Library/Formula/llvm.rb

```diff
@@ -153,9 +237,6 @@ def install
     end
 
     if build.with? "clang"
-      system "make", "-C", "projects/libcxx", "install",
-        "DSTROOT=#{prefix}", "SYMROOT=#{buildpath}/projects/libcxx"
-
```

Comment:
LLVM automatically detects the presence of libcxx, so I thought there isn't a need to make it manually.